### ISSUE TITLE
Add SNMP alerts

### ISSUE THREAD

**Issue by squareatom** [arachnys/cabot#185]
Please add ability to send SNMP alerts.

**Comment by dbuxton** [comment]
@squareatom this isn't something we would use but it should be reasonably trivial to write a plugin along the lines of https://github.com/bonniejools/cabot-alert-hipchat to do this.

@bonniejools is there documentation somewhere about how to specify plugins in config?

**Comment by bonniejools** [comment]
@dbuxton on the gh-pages branch of my fork, I've written some preliminary documentation on the process. I'll make a PR when it's finished but that's not for a while.

**Comment by dbuxton** [comment]
Shouldn't be hard to at least have something?

**Comment by bonniejools** [comment]
@dbuxton yeah. The 'for a while' was actually just figure of speech. It's almost finished anyway.

**Comment by benroeder** [comment]
https://github.com/trehn/hnmp Makes them easy to write

**Comment by xinity** [comment]
Why not using snmp enabled collector like collectd ? Snmp then would be a graphite metrics, easier to pull and triggger. My 2cents

**Comment by squareatom** [comment]
@benroeder @xinity I think you misunderstand. I'm requesting the ability to send SNMP traps from Cabot as a notification mechanism. NOT the ability to poll via SNMP. Thanks for hnmp though. I might use that for something else.
